doc/classes/ScrollContainer.xml

```diff
@@ -54,9 +54,15 @@
 		<member name="scroll_horizontal" type="int" setter="set_h_scroll" getter="get_h_scroll" default="0">
 			The current horizontal scroll value.
 		</member>
+		<member name="scroll_horizontal_custom_step" type="float" setter="set_horizontal_custom_step" getter="get_horizontal_custom_step" default="-1.0">
+			Overrides the step used when clicking the horizontal increment and decrement buttons or when using arrow keys when the [ScrollBar] is focused.
```

Comment:
```suggestion
			Overrides the step used when clicking the scroll bar's horizontal increment and decrement buttons or when using arrow keys when the [ScrollBar] is focused.
```
The buttons are not part of the container, so it's better to clarify that.
Same below.

Reply:
Good idea. Should I adapt the description of the class `ScrollBar` then too? 
It has nearly the same description:
`Overrides the step used when clicking increment and decrement buttons or when using arrow keys when the [ScrollBar] is focused.`

Reply:
No, these descriptions shouldn't be in sync. But ScrollBar's description of custom step should be improved.
ScrollBar's description mentions increment/decrement buttons, but they don't exist in the default theme. I think it could be clarified that you need to define their icons in the theme to make the buttons appear.
ScrollContainer uses ScrollBars as internal node and the buttons are part of ScrollBars, not ScrollContainer (I mean not directly). Its description could link to custom step in ScrollBar class, like
```
Overrides the [member ScrollBar.custom_step] of the internal scroll bar.
```

Reply:
> But ScrollBar's description of custom step could be improved.

Agree. What I mean is that it might be good to document in `ScrollBar` that the increment/decrement button are also internal, or at least kind of. Maybe a good idea for another PR though.

I adapted the doc changes. :)